src/planscape/planning/serializers.py

```diff
@@ -34,26 +34,40 @@ class Meta:
         geo_field = "geometry"
 
 
+class ScenarioResultSerializer(serializers.ModelSerializer):
+    class Meta:
+        fields = (
+            "id",
+            "created_at",
+            "updated_at",
+            "started_at",
+            "completed_at",
+            "status",
+            "result",
+            "run_details",
+        )
+        model = ScenarioResult
+
+
 class ScenarioSerializer(serializers.ModelSerializer):
     configuration = JSONField()
     notes = CharField(required=False)
     updated_at = DateTimeField(required=False)
     created_at = DateTimeField(required=False)
+    results = ScenarioResultSerializer(
+        required=False,
+        read_only=True,
+    )
 
     class Meta:
         fields = (
             "id",
+            "updated_at",
+            "created_at",
             "planning_area",
             "name",
             "notes",
             "configuration",
-            "updated_at",
-            "created_at",
+            "results",
```

Comment:
I'm wondering if we want to name this field something like "scenario_result" instead of just "results". It could be a little easy to confuse with the actual "result" of a ScenarioResult.

This isn't a super big deal if you'd rather just keep it as "results" though